src/browserClip.ts

```diff
@@ -111,13 +111,11 @@ const recursivelyApplyPatternToDurations = (
   return durations;
 };
 
-/**
- * @param  {Object}
- * @return {Tone.js Sequence Object}
- * Take a object literal that may have a Tone.js player OR instrument
- * or simply a sample or synth with a pattern and return a Tone.js sequence
- */
-module.exports = (params: ClipParams) => {
+const isToneLegacy = () => {
+  return Tone.version.split('.')[0] === '13';
```

Comment:
Since this is going for legacy we could even go for `return +Tone.version.split('.')[0] < 14;`

Reply:
That's right. I'll stick with `isToneV13` because I don't know the behavior for earlier versions of Tone.js.

Reply:
Solved in https://github.com/scribbletune/scribbletune/pull/113/commits/1ff91750cb3dad5b79bff216af47fcc83b9f781c